test/e2e/util.go

```diff
@@ -58,6 +61,19 @@ func clusterIPService(name, namespace string) *corev1.Service {
 	return testService(name, namespace, corev1.ServiceTypeClusterIP)
 }
 
+func testRoute(name, namespace string) *routev1.Route {
+	return &routev1.Route{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace:   namespace,
+			Name:        name,
+			Annotations: map[string]string{},
```

Comment:
Annotation has to be the same as one from the annotation filter of ExternalDNS instance.